gatsby-ssr.js

```diff
@@ -6,23 +6,26 @@ import ErrorBoundary from './src/sdk/error/ErrorBoundary'
 import Layout from './src/views/Layout'
 import TestProvider from './src/sdk/tests'
 import { validateCart } from './src/sdk/cart/validate'
+import AnalyticsHandler from './src/sdk/analytics'
 import { uiInitialState, uiActions, uiEffects } from './src/sdk/ui'
 
 export const wrapRootElement = ({ element }) => (
   <ErrorBoundary>
-    <TestProvider>
-      <UIProvider
-        initialState={uiInitialState}
-        actions={uiActions}
-        effects={uiEffects}
-      >
-        <SessionProvider>
-          <CartProvider mode="optimistic" onValidateCart={validateCart}>
-            {element}
-          </CartProvider>
-        </SessionProvider>
-      </UIProvider>
-    </TestProvider>
+    <AnalyticsHandler>
```

Comment:
Why not use the standard and call it `AnalyticsProvider` ?

Reply:
I agree it would 'match' with the other providers being used, but on a previous comment made by @icazevedo he pointed out that given the component implementation, the name "Provider" would not suit it well. I've tagged you in the conversation thread.